examples/tracetest-signalfx/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
 
     tracetest:
-        image: kubeshop/tracetest:v0.6.5
+        image: kubeshop/tracetest:v0.6.8
```

Comment:
If you use a version here, you won't be testing the actual code being built. you need to do a `build` or use the `sha-*` tag for the commit. You already depend on the `build docker` step, so that makes more sense

Reply:
I'm replacing those values before. I just bumped the values just to make sure if a user uses those examples, they are using a more stable version.

Reply:
You can use env vars. It would be a better apprach, because it's easier to read, very obvious you're replacing the value, and we don't need to maintain the examples:

https://docs.docker.com/compose/environment-variables/

Reply:
tried to use `compose run` but hit some problems. The easiest way was to generate a `.env` file and use it instead.

Reply:
Sorry for the confusion, my suggestion wasn't to use `run` instead of `up`. You don't need a `.env` file, you can set the env, or prepend the value to the cmd, like `TAG=v0.1.2 docker compose up`